### PR TITLE
Add a pip3_import target for Python 3 compatibility.

### PR DESCRIPTION
**Pull request by jkinkead**
Fix / workaround for #33 .

Much thanks to @joshclimacell for doing essentially this.

### CODE REVIEW

File: python/pip.bzl [hunk 1]
```diff
@@ -34,6 +34,14 @@ def _pip_import_impl(repository_ctx):
   if result.return_code:
     fail("pip_import failed: %s (%s)" % (result.stdout, result.stderr))
 
+def _pip2_import_impl(repository_ctx):
```
Comment (joshclimacell) [inline python/pip.bzl]: Not sure, but maybe there should be three options? `pip_import` which uses system `python`, `pip2_import` which explicitly uses `python2`, and `pip3_import` which explicitly uses `python3`?

File: python/pip.bzl [hunk 2]
```diff
@@ -13,7 +13,7 @@
 # limitations under the License.
```
Comment (joshclimacell) [inline python/pip.bzl]: Would probably be nice to add documentation of this functionality to a `README.md`?

Comment (jkinkead) [reply python/pip.bzl]: Docs updated.

File: python/pip.bzl [hunk 3]
```diff
@@ -24,16 +24,24 @@ def _pip_import_impl(repository_ctx):
...
   # To see the output, pass: quiet=False
   result = repository_ctx.execute([
-    "python", repository_ctx.path(repository_ctx.attr._script),
+    binary_name, repository_ctx.path(repository_ctx.attr._script),
     "--name", repository_ctx.attr.name,
     "--input", repository_ctx.path(repository_ctx.attr.requirements),
     "--output", repository_ctx.path("requirements.bzl"),
     "--directory", repository_ctx.path(""),
-  ])
+  ], quiet=False)
```
Comment (mishas) [inline python/pip.bzl]: I think you want to return it to quiet.

**Comment by variantf** [comment]
any timeline for this pr? really need this feature bring in!!!

**Comment by aptbosox** [comment]
I could really use this. Any updates?

**Comment by joshclimacell** [comment]
Not that I know of... I've been using this alternative Python packaging mechanism lately, which I like a lot: https://github.com/TriggerMail/rules_pyz

**Comment by marczych** [comment]
+1. I cherry-picked this and it works like a charm!

**Comment by akashgangil** [comment]
Can we please get this in?

**Comment by uri-canva** [comment]
@brandjon who is looking at PRs now?

**Comment by brandjon** [comment]
I should have some time for PR reviews after the cut for Bazel 0.25, which is April 1st. There's a lot of backlog in rules_python I'd like to work down starting then. We've been prioritizing cleanups to the native rules but this issue is certainly highly requested.

**Comment by prestonvanloon** [comment]
Any update on this?

**Comment by brandjon** [comment]
Update: We're using an approach based on #158, rebased and merged as #252. See #256 for the `pip3_import` macro.

**Pull request closed without merging**